### ISSUE TITLE
Project env vars in config file not being passed to parent terragrunt.hcl

### ISSUE THREAD

**Issue by Rwwn** [infracost/infracost#2579]
I am trying to set up Infracost on our Terragrunt repository. This repo keeps things DRY by re-using the same directory for different environments. Each environment is ran by specifying some environment variables (ENV and REGION). These values are found by Terragrunt using the get_env() function:

```hcl
locals {
  region = get_env("REGION")
  env    = get_env("ENV")
}
```

The above code is in in the "parent" terragrunt.hcl, which is included in the base terragrunt.hcl with an include block:

```hcl
include {
  path   = find_in_parent_folders()
  expose = true
}
```

The base terragrunt.hcl is the one Infracost is running against.

It seems that Infracost should be able to handle this by using a config file, where each environment is configured as its own project with its own set of env vars, according to the docs [here](https://www.infracost.io/docs/features/config_file/).

Example of my config file infracost.yml (TG_DIR is set to the directory I want to run Infracost in; the location of the base terragrunt.hcl):
```yaml
version: 0.1
projects:
    - name: dev
      path: ${TG_DIR}
      env:
        ENV: dev
        REGION: us-east-1
    - name: qa
      path: ${TG_DIR}
      env:
        ENV: qa
        REGION: us-east-1
    - name: prod
      path: ${TG_DIR}
      env:
        ENV: prod
        REGION: us-east-1
```

However, this doesn't work as expected. When running `infracost breakdown --config-file ${TG_DIR}/infracost.yml` it complains that the environment variables aren't defined:
```
Error in function call; Call to function "get_env" failed:
          EnvVarNotFound:
            Required environment variable ENV - not found.
```

It works fine when I don't use the config file but instead pass through the env vars when running the `infracost breakdown` command, but then I'd have to run each project separately, which isn't ideal. Is this a bug, or am I doing something wrong?

**Comment by Rwwn** [comment]
Thanks @aliscott  👍

**Comment by hugorut** [comment]
Hi @Rwwn, thanks for reporting this issue. We've released a fix for this in Infracost version [v0.10.28](https://github.com/infracost/infracost/releases/tag/v0.10.28)
